index.html

```diff
@@ -7,28 +7,32 @@
 <link rel="stylesheet" href="style.css" type="text/css">
 </head>
 <body>
+  <section class="resume">
   <header>
     <h1>Edgar Gasca Rodriguez</h1>
-    <aside>(717)-481-0806 edgargascarodriguez@gmail.com</aside>
+    <aside>(717)-481-0806 <a href="edgargascarodriguez@gmail.com">edgargascarodriguez@gmail.com</a></aside>
   </header>
   <article>
     <section class="section1">
-      <p></p>
+      <h2>Objectives/Goals</h2>
+      <p>My objective is to become a programmer and to succeed with a good education.</p>
     </section>
     <section class="section2">
-      <div></div>
-      <div></div>
+      <div><h2>Education</h2><p>I graduated in 2017 from Solanco Highschool, located in Quarryville PA. I received my highschool degree. I earned college credits when I was in highschool. I ended highschool with a GPA of 3.5. I am also currently enrolled at Thaddeus Stevens for software engineering.</p>
+</div>
+      <div><h2>Skills</h2><p>I am a very cooperative person. I work well with teams. I speak both Spanish and English fluently. I have a high charisma. I am easy to get along with and I communicate well with others. I show respect, listen well to others and treat everyone equally. I am a person that likes to get work done and support others when needed.</p></div>
```

Comment:
Indent issues. Do command a and then re-indent your lines while in your text editor